### PR TITLE
Feature/usage control for 3 0 0

### PR DESCRIPTION
**Pull request by sebbader**
Major rework of codes for LeftOperands, BinaryOperators, RightOperand datatypes.

### CODE REVIEW

File: model/contract/Rule.ttl [hunk 1]
```diff
@@ -150,6 +150,13 @@ ids:rightOperand rdfs:subPropertyOf odrl:rightOperand;
     rdfs:domain ids:Constraint;
     rdfs:range rdfs:Literal;
     rdfs:comment "The value of the right operand in a constraint expression."@en.
+	
+ids:rightOperandReference rdfs:subPropertyOf odrl:rightOperandReference;
+    a owl:ObjectProperty;
+    rdfs:label "rightOperandReference"@en ;
+    rdfs:domain ids:Constraint;
+    #rdfs:range rdfs:Resource;
```
[the remark this answers is not in the document]
Comment (sebbader) [reply model/contract/Rule.ttl]: In that case, I don't see the difference between an ids:rightOperand (line 147) and ids:rightOperandReference. Actually, how came up with rightOperandReference? Wasn't me... But makes sense somehow. 
`ids:rightOperand rdfs:range rdfs:Literal;` doesn't look good, neither does `rdfs:range xsd:string`...

[the remark this answers is not in the document]
Comment (sebbader) [reply model/contract/Rule.ttl]: Having both allows us to link to Literals and Entities

[the remark this answers is not in the document]
Comment (HaydarAk) [reply model/contract/Rule.ttl]: Seems like it was added some time ago. 3c129472ab903cde024e34026a5982b1d647f008 (odel/contract/Rule.ttl)

---

From a "semantic" perspective it makes sense to have both:

- `ids:rightOperand` to do assign values directly, e.g., to say 
idsc:COUNT idsc:LT 10

- `ids:rightOperandReference` with xsd:string to assign an IRI as the rightOperand, which has to be deferenced to get the actual value.

Based on our ids:BinaryOperators, I can see different possibilities for right operands.
- date / datetime
- integer/double values for count-based access or payment amounts
- strings
- endpoints (by IRI?) to retrieve information, e.g. as suggested in the `idsc:MEMBER_OF` operator
- Entities like ids:TemporalEntity and it's subclasses, ids:SpatialEntity and it's subclasses for time/spatial constraints
___ 

In my opinion, we should stick to the existing modelling. 
Having `ids:rightOperand` with range rdfs:Literal could be used in all cases, where only simple values have to be provided as right operands such as the first 4 elements mentioned above..  Using `ids:rightOperandReference` with range rdfs:Resource can be used for RDF modelled constructs, such as ids:TemporalEntity / ids:Interval / ids:SpatialEntity and so on. 

#### My suggestion:  
Uncomment `rdfs:range rdfs:Resource` 
```
ids:rightOperandReference rdfs:subPropertyOf odrl:rightOperandReference;
    a owl:DatatypeProperty;
    rdfs:label "rightOperandReference"@en ;
    rdfs:domain ids:Constraint;
    rdfs:range rdfs:Resource
```

**Comment by sebbader** [comment]
closing in favor of #210

**Pull request closed without merging**